packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IPersona, Persona, Stack, PersonaSize, Text, mergeStyles } from '@fluentui/react';
+import React, { useEffect, useRef } from 'react';
+import { _pxToRem } from '@internal/acs-ui-common';
+import { _FileUploadCardsStrings } from './FileUploadCards';
+import { OnRenderAvatarCallback } from '../types';
+import { Ref } from '@fluentui/react-northstar';
+
+/**
+ * @internal
+ * information required for each line of caption
+ */
+export type CaptionInfo = {
+  displayName: string;
+  caption: string;
+  userId?: string;
+};
+
+/**
+ * @internal
+ * _CaptionsBanner Component Props.
+ */
+export interface _CaptionsBannerProps {
+  captions: CaptionInfo[];
+  /**
+   * Optional callback to override render of the avatar.
+   *
+   * @param userId - user Id
+   */
+  onRenderAvatar?: OnRenderAvatarCallback;
+}
+
+/**
+ * @internal
+ * A component for displaying a CaptionsBanner with user icon, displayName and captions text.
+ */
+export const _CaptionsBanner = (props: _CaptionsBannerProps): JSX.Element => {
+  const { captions, onRenderAvatar } = props;
+
+  const gridContainerClassName = mergeStyles({
+    overflowY: 'scroll',
+    overflowX: 'hidden',
+    width: '100%',
+    height: _pxToRem(60),
+    display: 'grid',
+    gridTemplateColumns: '20% 80%',
+    alignItems: 'stretch',
+    columnGap: _pxToRem(16),
+    padding: _pxToRem(8)
+  });
+
+  const displayNameClassName = mergeStyles({
+    fontWeight: 600,
+    fontSize: _pxToRem(12),
+    lineHeight: _pxToRem(30)
+  });
+
+  const captionClassName = mergeStyles({
+    fontWeight: 400,
+    fontSize: _pxToRem(16),
+    lineHeight: _pxToRem(30)
+  });
+
+  const captionsScrollDivRef = useRef<HTMLElement>(null);
+  const scrollToBottom = (): void => {
+    if (captionsScrollDivRef.current) {
+      captionsScrollDivRef.current.scrollTop = captionsScrollDivRef.current.scrollHeight;
+    }
+  };
+
+  useEffect(() => {
+    scrollToBottom();
+  }, [captions]);
+
+  return (
+    <Ref innerRef={captionsScrollDivRef}>
+      <div data-is-focusable={true} className={gridContainerClassName}>
+        {captions.map((caption) => {
```

Comment:
Would be nice to give these caption a key to improve perf:
https://60c7ae6891f0e90039d7cd54-ylgsirfuaq.chromatic.com/?path=/docs/ui-components-internal-captionsbanner--captions-banner
Otherwise react will try to render all the list everytime (It's fine for 50 instance TBH, but why not?)

Reply:
Posted a wrong link, this is the right one:
https://reactjs.org/docs/lists-and-keys.html#keys